xpack_security_get_user.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2012-2019 Oliver Eilhard. All rights reserved.
+// Use of this source code is governed by a MIT-license.
+// See http://olivere.mit-license.org/license.txt for details.
+
+package elastic
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/url"
+
+	"github.com/olivere/elastic/v7/uritemplates"
+)
+
+// XPackSecurityGetUserService retrieves a user by its name.
+// See https://www.elastic.co/guide/en/elasticsearch/reference/7.0/security-api-get-user.html.
+type XPackSecurityGetUserService struct {
+	client *Client
+	pretty bool
+	name   string
```

Comment:
The `name` is not a valid parameter for this API. See the [REST API spec](https://github.com/elastic/elasticsearch/blob/7.4/x-pack/plugin/src/test/resources/rest-api-spec/api/security.get_user.json) for a list of parameters: It should be one or more `username`.

Reply:
Right, I have added username as a list of strings. Are you ok with that ?